README.md

```diff
@@ -32,8 +34,7 @@ support privileged extension, root installation, or local repositories.
 
 ## Building and Installing
 
-Android SDK path should be specified either using `ANDROID_HOME` environment variable or using `sdk.dir` property in
-`local.properties`.
+To join the development, specify your Android SDK path either using the `ANDROID_HOME` environment variable, or by filling out the `sdk.dir` property in `local.properties`.
```

Comment:
"To join the development" is not correct. What should they do if they don't want to join the development, but just want to build it?

The purpose of this block is to describe how to build and install the package, that's all. It's important because GPL actually requires such explanation, so it's placed into readme.

As I said last time reviewing this block, I prefer my version, because this part is purely technical, and shouldn't imply anything.

Reply:
Also, I limit all lines in this file to 120 characters.